pkg/controller/enrollment/controller.go

```diff
@@ -19,8 +19,10 @@ var (
 
 	// DefaultOptions return an Options with default values filled in.
 	DefaultOptions = enrollment.Options{
-		SyncInterval:       types.Duration(5 * time.Second),
-		DestroyOnTerminate: false,
+		SyncInterval:         types.Duration(5 * time.Second),
+		DestroyOnTerminate:   false,
+		SourceParseErrOp:     enrollment.SourceParseErrorEnableDestroy,
```

Comment:
Tiny nit:  how do you feel about naming this `SourceParseErrPolicy` and `EnrollmentParseErrPolicy`?

Reply:
Sure, that's trivial. I'll make that change.